CONTRIBUTING.md

```diff
@@ -36,11 +36,11 @@ docker-compose up broker
 
 # Start node
 bin/server_build
-bin/server_run --node 123456
+bin/server_run --identity=0x123456..
 
 # Client connects to node
 bin/client_build
-bin/client_run --node 123456
+bin/client_run --cli
```

Comment:
This won't start client itself - please use `bin/cli_client_run`or run client separately.

Reply:
Fixed